ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -103,6 +103,11 @@ public String getVersion() {
             linearOptimisationEngine.run(linearRaoData, linearRaoParameters);
             LOGGER.info("Iteration {} - linear optimization [end]", iteration);
 
+            if (!linearOptimisationEngine.getSolverResultStatusString().equals("OPTIMAL")) {
+                LOGGER.info("Iteration {} - linear optimization is infeasible", iteration); //handle INFEASIBLE solver status
+                break;
+            }
+
```

Comment:
Why did you add this check? Is it a behaviour specific to loopflow computation?

Reply:
Non, it is not specific to loopflow computation.